install.sh

```diff
@@ -297,6 +297,49 @@ ops_link() {
   fi
 }
 
+checkHWAccelSupport() {
+  echo ""
+
+  local hwSupported=false
+  local haveRights=false
+
+  if [ "$OS" = "linux" ]; then
+    local group="kvm"
+    local username=`whoami`
+    local hw_support="grep -woE 'svm|vmx' /proc/cpuinfo | uniq"
+
+    #check acceleration supported
+    if [ "$hw_support" = "svm" ] || [ "$hw_support" = "vmx" ]; then
+      hwSupported=true
+    fi
+
+    #check permissions
+    if groups $username | grep -q "\b${group}\b"; then
+        haveRights=true
+    fi
+  fi
+
+  if [ "$OS" = "darwin" ]; then
+    haveRights=true
+    local hw_support=`sysctl kern.hv_support`
+    if [ "$hw_support" = "kern.hv_support: 1" ]; then
+      hwSupported=true
+    fi
+  fi
+
+  if [ "$hwSupported" = "false" ]; then
+        echo "$yellow> Hardware acceleration not supported by your system. $reset"
+        echo "$yellow> Ops will attempt to enable acceleration by default and will show a warning if the system doesn't support it. $reset"
+        echo "$yellow> To avoid such warnings you may disable acceleration in configuration or via command line parameters. $reset"
+  else
+    if [ "$haveRights" = "false" ]; then
+      echo "$yellow> Hardware acceleration is supported, but you don't have rights. $reset"
+      echo "$yellow> Try adding yourself to the kvm group: sudo adduser ${username} kvm $reset"
+      echo "$yellow> You'll need to re-login for this to take affect. $reset"
```

Comment:
Replace "affect" with "effect" (I know this spelling error was there before, but here's an opportunity to correct it).

Reply:
done